requirements.txt

```diff
@@ -0,0 +1,29 @@
+attrs==21.2.0
+cycler==0.10.0
+docutils==0.17.1
+-e git+https://github.com/EMVA1288/emva1288.git@0c88800ea20e4713f40c3ef9387b5e1f96d5317b#egg=emva1288
```

Comment:
If user is cloning the emva library, it should be installed from requirements.txt file. The user should use setup.py to do that directly (pip install -e.).

Reply:
I have created a requirements-test.txt for the additional test packages portion only. The user can continue use setup.py directly in place.